profiles/fscloud/variables.tf

```diff
@@ -0,0 +1,161 @@
+##############################################################################
+# Input Variables
+##############################################################################
+
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to use, set via environment variable TF_VAR_ibmcloud_api_key"
```

Comment:
remove the text `set via environment variable TF_VAR_ibmcloud_api_key` as this doesn't make sense in a schematics or projects world